### PR TITLE
Added ignore_group_order argument

### PR DESCRIPTION
**Pull request by efriman**
Changed how flip_negative_strand is implemented
Added ignore_group_order argument, which combines groups (e.g. group1-group2 and group2-group1)
Changed how coordinates are added when using store_stripes
Fixed rescaling with zeros
Added error msgs
Updated walkthroughs
Added test for stripes and ignorer_group_order
Fixed Readthedocs error with urllib3
Fixed misnamed rows/cols info in plotpup.py
Changed divide_pups to not error when columns are non-identical, warn instead

### CODE REVIEW

File: coolpuppy/coolpup.py [hunk 1]
```diff
@@ -1190,6 +1190,20 @@ def _stream_snips(self, intervals, region1, region2=None):
                 snip["data"] = np.rot90(np.flipud(snip["data"]))
                 if self.expected and not self.ooe:
                     exp_data = np.rot90(np.flipud(exp_data))
+            
+            if self.ignore_group_order:
+                groups = [g for g in groupby if g not in ["strand1", "strand2", "distance_band"]]
+                values_sorted = sorted([snip[groups[0]], snip[groups[1]]])
```
Comment (Phlya) [inline coolpuppy/coolpup.py:1196]: What is there are not exactly 2 groups?

**Comment by efriman** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/open2c/coolpuppy/pull/130/discussion/#comment-1620382454'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by Phlya** [comment]
<p>Very nice, thanks!</p><p> </p>

---

 <a href='https://app.reviewnb.com/open2c/coolpuppy/pull/130/discussion/#comment-1620382454'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by Phlya** [comment]
Can you add a test for ignore_group_order argument? Otherwise looks good to me, thank you!

**Comment by efriman** [comment]
> Can you add a test for ignore_group_order argument? Otherwise looks good to me, thank you!

Done :)

**Comment by Phlya** [comment]
Fantastic, I think it's ready to merge, do you have anything else to add here or is it done from your side?

**Pull request merged**